tools/test-proxy/Azure.Sdk.Tools.TestProxy/Store/GitStore.cs

```diff
@@ -209,6 +212,13 @@ public string[] DetectPendingChanges(GitAssetsConfiguration config)
         /// <param name="config"></param>
         public void CheckoutRepoAtConfig(GitAssetsConfiguration config)
         {
+            if (Assets.TryGetValue(config.AssetsJsonRelativeLocation, out var value)){
+                if (value == config.Tag)
+                {
+                    return;
+                }
+            }
```

Comment:
nit: curly brackets on their own line and the conditions can be merged:

```suggestion
            if (Assets.TryGetValue(config.AssetsJsonRelativeLocation, out var value) && value == config.Tag)
            {
                return;
            }
```

Reply:
Yeah, it'll only eval the right if left passes, so that should definitely work.